README.md

```diff
@@ -1,23 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Portfolio
 
-> One paragraph statement about the project.
+> This is the first project of my portfolio. It has no functionally as yet.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> This project was to show my knowledge of what I have learned on HTML, CSS, and using flexbox.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
 
 ## Getting Started
```

Comment:
- [OPTIONAL ] Hi, nice one implementing the required changes. However, consider adding a screenshot to your README because README is the first place visitors to your repository go to first. Adding descriptive README with images of your work attracts visitors and makes them want to checkout your work.

- [OPTIONAL ] Consider adding these changes below the `Getting Started` :

   ### To get a local copy up and running follow these simple steps.
  - You can clone this repo by typing `git clone (https://github.com/kemar-art/My-Portfolio.git)` on your terminal
  - Type `cd My-Portfolio` to access the project on your terminal`

- [ OPTIONAL] To make your readme look professional, you can consider adding more information about the project and deleting other unnecessary lines 🤗 .

Example: All the content in the image below is supposed to be replaced by the description written by you.

<img width="604" alt="chrome_cmNzTKd4M3" src="https://user-images.githubusercontent.com/81468675/144309313-8b6c6090-f53e-4d78-9f9e-1022dd2df98a.png">